model/src/test/kotlin/config/PackageConfigurationTest.kt

```diff
@@ -159,6 +160,45 @@ class PackageConfigurationTest : WordSpec({
             ) shouldBe true
         }
 
+        "return true if artifact source code origin and identifier are equal, and version is in range" {
```

Comment:
maybe: "return true if source code origin is equal and identifier matches with version is in range"

(because `Identifier` instances are not equal, due to the use of version range)